### PR TITLE
Revising virtualization configuration topic

### PR DESCRIPTION
**Pull request by MayaKirova**
Related to bugs 232322 and 232674

### CODE REVIEW

**Review by mpavlinov — CHANGES_REQUESTED** [review]
1. Please add a separate section for column virtualization. If the developer is particularly interested to configure column virtualization he will search for such section.
2. Add a link to the "Feature Compatibility Matrix" in the "Related Topics" section.
3. Add new section "Enabling and Configuring Column Virtualization". Put it after the continuous virtualization section.
4. Remove "Enabling Virtualization Overview" section. It basically describes how to enable column virtualization.
5. In "Enabling and Configuring Fixed Virtualization" section talk about how to enable only fixed row virtualization. Remove any information about the column virtualization. Leave one note that links to column virtualization.
6.In "Enabling and Configuring Continuous Virtualization" section property settings are not correct and the example is not correct. Remove the "GetGlobalResourceObject" from the sample.

**Comment by kdinev** [comment]
@MayaKirova make it more explicit that the virtualization feature is usable in very specific scenarios when you cannot turn on other paging features because of an application requirement, and the size of the data is mid-range (1000-10000 records).

Also you can directly say that it's advisable to try other paging features when the application utilizes large amounts of data, before trying virtualization. Also mention that for small data sets, the virtualization feature will actually cause a performance hit, rather than a performance boost, due to the re-rendering of the virtual dom.

**Comment by kdinev** [comment]
@mpavlinov Would you approve with the latest changes, or does the topic require more changes?

**Review by mpavlinov — APPROVED** [review]

**Comment by mpavlinov** [comment]
@kdinev: I made some changes to the topics. Please, review them.

File: topics/02_Controls/igGrid/03_Features/13_Virtualization/00_igGrid_Virtualization_Overview.md [hunk 1]
```diff
@@ -10,74 +10,100 @@
...
+	-   [Continuous Row Virtualization](#continuous)
+-   [**Keyboard Interactions**](#keyboard-interactions)
+-   [**Related Content**](#related-content)
+    -   [Topics](#topics)
+	-   [Samples](#samples)
 
+## <a id="introduction"></a> Introduction
 
-## Introduction
+Virtualization is a feature of the `igGrid` that boosts performance when displaying large data sets containing thousands of records. It improves both rendering time and scrolling as well as memory footprint. It works by reducing the number of the DOM objects in-memory and reusing them while user scrolls and operates with the data.
```
Comment (kdinev) [inline topics/02_Controls/igGrid/03_Features/13_Virtualization/00_igGrid_Virtualization_Overview.md]: The topic still doesn't state that this feature is not the feature of choice if you can enable, for example, paging. Also I didn't find anything related to the optimal number of records this feature is suitable for.

**Comment by kdinev** [comment]
You can also add the information that virtualization is really optimal for a scenario with 1000-10000 records all displayed in one page.

**Review by kdinev — CHANGES_REQUESTED** [review]
I left a comment above.

**Comment by mpavlinov** [comment]
@kdinev I applied your comments. Please, review.

**Review by kdinev — APPROVED** [review]

**Pull request merged**